src/status_im2/contexts/quo_preview/selectors/disclaimer.cljs

```diff
@@ -1,27 +1,49 @@
 (ns status-im2.contexts.quo-preview.selectors.disclaimer
   (:require [quo2.components.buttons.button :as button]
-            [quo2.components.selectors.disclaimer.view :as quo]
+            [quo2.components.selectors.disclaimer.view :as disclaimer]
             [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
             [react-native.core :as rn]
-            [reagent.core :as reagent]))
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label "Checked:"
+    :key   :checked?
+    :type  :boolean}
+   {:label "Blur (only for dark theme):"
+    :key   :blur?
+    :type  :boolean}])
 
 (defn cool-preview
   []
-  (let [checked? (reagent/atom false)]
+  (let [state (reagent/atom {:checked? false
+                             :blur?    true})]
     (fn []
-      [rn/view
-       {:margin-bottom      50
-        :padding-vertical   16
-        :padding-horizontal 20}
-       [rn/view
-        {:padding-vertical 60
-         :align-items      :center}
-        [quo/view
-         {:container-style {:margin-bottom 40}
-          :on-change       #(swap! checked? not)}
-         "I agree with the community rules"]
-        [button/button
-         {:disabled (not @checked?)}
+      [rn/view {:style {:flex 1}}
+       [rn/view {:style {:flex 1}}
+        [preview/customizer state descriptor]]
+       [rn/view {:style {:padding-horizontal 15}}
+        (when (and (:blur? @state) (theme/dark?))
+          [rn/view
+           {:style {:position :absolute
+                    :top      0
+                    :bottom   0
+                    :left     0
+                    :right    0}}
+           [preview/blur-view
+            {:style                 {:flex        1
+                                     :align-items :center}
+             :show-blur-background? true}]])
+        [rn/view
+         {:style {:margin-vertical 50
+                  :width           "100%"}}
+         [disclaimer/view
+          {:blur?     (:blur? @state)
+           :checked?  (:checked? @state)
+           :on-change #(swap! state update :checked? not)}
+          "I agree with the community rules"]]
```

Comment:
`i18n`
`:t/agree-terms` ?

Reply:
Oh, this is a preview screen. I thought it was not important if we use strings without a localization for these screens.

Reply:
Yeah we don't need i18n for preview screens 👍

Reply:
Thinking more about this, that text should be a paramete available in the preview screen in order to test it (I already added the commit):

![image](https://user-images.githubusercontent.com/90291778/227379344-f3d62626-c634-4668-b36b-0dd1052ca36e.png)